vector/v.net.path/v.net.path.html

```diff
@@ -200,8 +200,3 @@ <h3>TURNS SUPPORT</h3>
 Implementation: Stepan Turek<br>
 Documentation: Lukas Bocan, Eliska Kyzlikova, Viera Bejdova<br>
 Mentor: Martin Landa
-
-<!--
-<p>
-<i>Last changed: $Date$</i>
--->
```

Comment:
Line 85, 177 unnecessary one empty line.

Line 87 missing one empty line.

Reply:
(it might be faster to edit it in the GH web editor right away with multiline editing :-) )